### PR TITLE
Updated Readme with new images

### PR DESCRIPTION
**Pull request by Abhay-dot**
As part of my [GCI task](https://codein.withgoogle.com/dashboard/task-instances/5420843031592960/) - Turtle Blocks Issue: Change the readme images to the new UI

I added a new Gif in the screenshots folder. I have a couple of doubts, do we have to remove the screenshot which are not used anymore.
For updating the icons which are present in the Readme should i take screenshots or what is the right approach.
@walterbender @vaibhavdaren @ezequielpereira

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ more adventurous student.
 > rather than the JavaScript version. (The JavaScript version is
 > included in Sugarizer.)
 
-![Turtle Blocks](https://github.com/sugarlabs/turtleblocksjs/blob/master/screenshots/updatedscreenshot.png "Turtle Blocks")
+![Turtle Blocks](https://github.com/Abhay-dot/turtleblocksjs/blob/updateReadme/screenshots/screenshot-2019-12-4%20Turtle%20Block%20JS.gif "Turtle Blocks")
```
Comment (walterbender) [inline README.md]: These need to be links to sugarlabs and master... once the PR is merged, it will work.

Comment (Abhay-dot) [reply README.md]: hey @walterbender i updated the code as you suggested

**Comment by Abhay-dot** [comment]
Hey,
	I updated as you recommended 

Thanks so much for helping me,
Abhay

**Comment by Abhay-dot** [comment]
Hey,
	so after updating the Readme file as you suggested do you think its good to go for the task review.

Thanks,
Abhay

**Pull request merged**